test/over_react/util/handler_chain_util_test.dart

```diff
@@ -158,7 +160,7 @@ main() {
             test('calls all functions in order', () {
               var calls = [];
 
-              var functions = new List.generate(5, (index) {
+              List<Function> functions = new List.generate(5, (index) {
```

Comment:
Ah, sorry, should it be this, then?
```dart
var functions = new List<Callback0Arg>.generate(5, (index) {
```

I'm fine leaving it as-is if it works, though